gipc/gipc.py

```diff
@@ -26,6 +26,7 @@
 import logging
 import multiprocessing
 import multiprocessing.process
+import multiprocessing.reduction as mpreduction
```

Comment:
I'm curious how you would prefer to import this

`multiprocessing.reduction.register()` is quite long... not sure if you have a preference but
`from multipprocessing import reduction`  would be a bit shorter

Reply:
I see you found the previous import : ).

Btw, I always like clarity over length concerns.